### PR TITLE
#trivial - Add readme info for non vue sites.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added

- Readme information on how to render f-searchbox in non-vue instances.

### CODE REVIEW

File: packages/components/molecules/f-searchbox/README.md [hunk 1]
```diff
@@ -227,3 +227,47 @@ const copyOverrides = {
...
+<head>
+    <title>f-searchbox example standalone</title>
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/f-searchbox/dist/f-searchbox.css">
+</head>
+<body>
+    <div data-app>
+        <Vue-Searchbox locale="en-GB" />
+    </div>
+    <script src="https://unpkg.com/vue@2.6.11/dist/vue.js"></script>
+    <script src="https://unpkg.com/vuex@3.2.0"></script>
```
Comment (DamianMullins) [inline packages/components/molecules/f-searchbox/README.md]: Might be best to link to the full URL here? - https://unpkg.com/vuex@3.2.0/dist/vuex.js

File: packages/components/molecules/f-searchbox/README.md [hunk 2]
```diff
@@ -227,3 +227,47 @@ const copyOverrides = {
...
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <title>f-searchbox example standalone</title>
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/f-searchbox/dist/f-searchbox.css">
+</head>
+<body>
+    <div data-app>
+        <Vue-Searchbox locale="en-GB" />
```
Comment (DamianMullins) [inline packages/components/molecules/f-searchbox/README.md]: Should use lower kebab-case here e.g. `vue-searchbox`

File: packages/components/molecules/f-searchbox/README.md (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/README.md]: good spot.

**Review by DamianMullins — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

File: packages/components/molecules/f-searchbox/README.md [hunk 3]
```diff
@@ -227,3 +227,47 @@ const copyOverrides = {
...
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <title>f-searchbox example standalone</title>
+    <link rel="stylesheet" href="https://unpkg.com/@justeat/f-searchbox/dist/f-searchbox.css">
+</head>
+<body>
+    <div data-app>
+        <vue-Searchbox locale="en-GB" />
```
Comment (kevinrodrigues) [inline packages/components/molecules/f-searchbox/README.md:253]: Ill lowercase the `S` in my current pr. missed it :\

Comment (DamianMullins) [reply packages/components/molecules/f-searchbox/README.md:253]: Ah yeah, I missed that too

Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/README.md:253]: Fixed https://github.com/justeat/fozzie-components/pull/651/files#diff-702ba200f743792e5005b0e1e0c5dd447c3569ca61dc0d29a6c0fde0408f44b7R253

**Pull request merged**